cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -226,16 +223,20 @@ def chargeback_costs_default(resource_usage, db, provider):
     average_disk_io = resource_usage['average_disk_io']
 
     cpu_rate = query_rate(db, provider, 'cpu_usagemhz_rate_average', 'Default', 'Compute')
-    cpu_used_cost = average_cpu_used_in_mhz * float(cpu_rate) / 24
+    cpu_used_cost = version.pick({version.LOWEST: average_cpu_used_in_mhz * float(cpu_rate),
+                            '5.7': average_cpu_used_in_mhz * float(cpu_rate) * 24})
 
     memory_rate = query_rate(db, provider, 'derived_memory_used', 'Default', 'Compute')
-    memory_used_cost = average_memory_used_in_mb * float(memory_rate) / 24
+    memory_used_cost = version.pick({version.LOWEST: average_memory_used_in_mb * float(memory_rate),
```

Comment:
compared to the other transformations this one looks the wrong way around wrt factors/divisors

there is more than one of those, please verify if this was a mistake before the transformation, or if this was lost in unification